Automattic/wp-calypso
# Full Site Editing: ignore `dist` directories

#### Changes proposed in this Pull Request

* add a `.gitignore` to ignore `dist` directories

#### Testing instructions

1. Build the project using the [README](https://github.com/Automattic/wp-calypso/blob/master/apps/full-site-editing/README.md) to build the plugin.
2. Run `git st` and notice that it now ignores the `dist` directories
